test/param/int-list.spec.js

```diff
@@ -0,0 +1,134 @@
+const expect = require('chai').expect;
```

Comment:
Much of this testing is already done in `NumberList`. So after you extend from `NumberList`, you will be able to remove a lot of these tests.